pkg/cmd/component/bootstrap_new.go

```diff
@@ -198,15 +206,15 @@ func initiateInteractiveModeForBootstrapNewCommand(io *BootstrapNewParameters, c
 		return err
 	}
 	if secret == "" { // We must prompt for the token
-		if io.Secret == "" {
-			io.Secret = ui.EnterGitSecret(io.GitRepoURL)
+		if io.Token == "" {
+			io.Token = ui.EnterGitSecret(io.GitRepoURL)
```

Comment:
Rename the EnterGitSecret function to EnterGitToken

Reply:
Done